resview.py

```diff
@@ -116,6 +116,12 @@ def make_cells_from_conn(conns, convert_to_vtk_type):
 
     return cells, cell_type, offset
 
+def add_mat_id_to_grid(grid, mesh):
```

Comment:
Calling the function with `mesh` is unnecessary, `cell_groups` array would be sufficient.